vdr/types/common.go

```diff
@@ -34,6 +34,9 @@ var ErrInvalidDID = errors.New("invalid did syntax")
 // ErrKeyNotFound is returned when a particular key or type of key is not found.
 var ErrKeyNotFound = errors.New("key not found")
 
+// ErrDIDNotManagedByThisNode is returned when an operation needs the private key and if is not found on this host
+var ErrDIDNotManagedByThisNode = errors.New("did document not managed by this host")
```

Comment:
I'm kinda allergic about using acronyms without capitals (did -> DID).

Also the message (host) differs from the name (node)

Reply:
errors cannot start with capitals?

Reply:
it can, but it is discouraged to make it a sentence (like "The DID Document is not managed by this node.") because it hampers error wrapping. I remember IntelliJ not complaining when you use an acronym (so starting with "DID document" should be fine).

Reply:
and you can silence codeclimate by hand ;)